src/main.cpp

```diff
@@ -1,6 +1,21 @@
 #include <iostream>
 
+#include "objects/disk.h"
+#include "objects/sphere.h"
+#include "render/renderer.h"
+
 int main()
 {
-    std::cout << "hello, world!" << std::endl;
+    render::Renderer renderer{ 90.0f, 150, 150 };
+    renderer.getScene().directionalLight = lmath::Vec3( 0.6f, -0.8f, 0.0f ).normalize();
```

Comment:
i think it's really strange to set something through a getter. maybe we need to make a setter for directional light and objects bellow. Methods like addObj() and setDirectionalLight().

Reply:
Yeah, it is kinda weird, but I made it in mind that we will be able to swap the scene on the fly. And I don't want the renderer class to have methods that have nothing to do with the render process itself. We'll look into it when we work on something more complex, I think then we'll come up with a better solution.